cli/rack/__init__.py

```diff
@@ -423,7 +425,17 @@ def go() -> dict:
             go()
         
         if manifest.getPerformOptimization():
-            logger.warning("Optimization requested but not yet implemented")
+            invoke_optimization(optimization_url)
+
+def invoke_optimization(url: Optional[Url]) -> None:
+    @with_status(f'Optimizing triplestore')
+    def go() -> None:
+        url = url or DEFAULT_OPTIMIZE_URL
+        response = requests.get(url).json()
+        if not response['success']:
```

Comment:
Just to double-check, the `'success'` field is populated (with something falsy) even on failure?

Reply:
Yes, it appears so